docker-compose.yml

```diff
@@ -71,6 +71,8 @@ services:
     build:
       context: ./test_experiment
       dockerfile: Dockerfile
+      platforms:
+        - "linux/amd64"
```

Comment:
Lets add a comment here 

- "linux/amd64" # once both boto3,mlfow will work on aarch64 (arm) remove this line

Reply:
or may be add `"once continuumio/miniconda3:latest image work on native aarch64 (arm) remove this line"` ? I think maybe the error is caused by this base image doesn't have work fine in my mac? Because I can easily just `pip install boto3 mlfow` in conda local, but failed in this dockerfile
```
FROM continuumio/miniconda3:latest
RUN pip install mlflow boto3
```

Reply:
perfect!

Reply:
finished!